Troubleshooting/heal-info-and-split-brain-resolution.md

```diff
@@ -0,0 +1,461 @@
+Heal info and split-brain resolution 
+=======================================
+This document explains the  heal info command available in gluster for monitoring pending heals in replicate volumes and the methods available to resolve split-brains.  
+
+# 1. Volume heal info:  
+Usage: `gluster volume heal <VOLNAME> info`  
+This lists all the files that need healing (and will be processed by the self-heal daemon). It prints either their path or their GFID.
+### Interpretting the output
+All the files that are listed in the output of this command need healing to be
+done.The files listed may also be accompained by the following tags:
+
+a) 'Is in split-brain'  
+A file in data or metadata split-brain will 
+be listed with " - Is in split-brain" appended after its path/gfid. Eg., 
+"/file4" in the output provided below. But for a file in gfid split-brain,
+ the parent directory of the file is shown to be in split-brain and the file 
+itself is shown to be needing heal. Eg., "/dir" in the output provided below 
+which is in split-brain because of gfid split-brain of file "/dir/a".
+Files in split-brain cannot be healed without resolving the split-brain.
+
+b) 'Is possibly undergoing heal'  
```

Comment:
Yup, it is printed. Source code @ https://github.com/gluster/glusterfs/blob/release-3.8/heal/src/glfs-heal.c#L481